pkg/operation/botanist/controlplane/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -0,0 +1,529 @@
+		"--pod-eviction-timeout=2m0s",
+		"--root-ca-file=/srv/kubernetes/ca/ca.crt",
+		"--service-account-private-key-file=/srv/kubernetes/service-account-key/id_rsa",
+		fmt.Sprintf("--service-cluster-ip-range=%s", k.serviceNetwork.String()),
+	)
+
+	if versionConstraintK8sGreaterEqual113.Check(k.version) {
+		command = append(command,
+			fmt.Sprintf("--secure-port=%d", port),
+			"--port=0",
+		)
+	}
+
+	if versionConstraintK8sGreaterEqual112.Check(k.version) {
+		command = append(command,
+			fmt.Sprintf("--horizontal-pod-autoscaler-downscale-stabilization=%s", defaultHorizontalPodAutoscalerConfig.DownscaleStabilization.Duration.String()),
+			fmt.Sprintf("--horizontal-pod-autoscaler-initial-readiness-delay=%s", defaultHorizontalPodAutoscalerConfig.InitialReadinessDelay.Duration.String()),
+			fmt.Sprintf("--horizontal-pod-autoscaler-cpu-initialization-period=%s", defaultHorizontalPodAutoscalerConfig.CPUInitializationPeriod.Duration.String()),
+			fmt.Sprintf("--authentication-kubeconfig=%s/%s", volumeMountPathKubeconfig, secrets.DataKeyKubeconfig),
+			fmt.Sprintf("--authorization-kubeconfig=%s/%s", volumeMountPathKubeconfig, secrets.DataKeyKubeconfig),
+			fmt.Sprintf("--tls-cert-file=%s/%s", volumeMountPathServer, secrets.ControlPlaneSecretDataKeyCertificatePEM(SecretNameServer)),
+			fmt.Sprintf("--tls-private-key-file=%s/%s", volumeMountPathServer, secrets.ControlPlaneSecretDataKeyPrivateKey(SecretNameServer)),
+		)
+	}
+
+	command = append(command,
+		"--tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
+		"--use-service-account-credentials=true",
+		"--v=2",
+	)
+	return command
+}
+
+func (k *kubeControllerManager) getHorizontalPodAutoscalerConfig() gardencorev1beta1.HorizontalPodAutoscalerConfig {
+	defaultHPATolerance := gardencorev1beta1.DefaultHPATolerance
+	horizontalPodAutoscalerConfig := gardencorev1beta1.HorizontalPodAutoscalerConfig{
+		CPUInitializationPeriod: &metav1.Duration{Duration: gardencorev1beta1.DefaultCPUInitializationPeriod},
+		DownscaleDelay:          &metav1.Duration{Duration: gardencorev1beta1.DefaultHPADownscaleDelay},
+		DownscaleStabilization:  &metav1.Duration{Duration: gardencorev1beta1.DefaultDownscaleStabilization},
+		InitialReadinessDelay:   &metav1.Duration{Duration: gardencorev1beta1.DefaultInitialReadinessDelay},
+		SyncPeriod:              &metav1.Duration{Duration: gardencorev1beta1.DefaultHPASyncPeriod},
+		Tolerance:               &defaultHPATolerance,
+		UpscaleDelay:            &metav1.Duration{Duration: gardencorev1beta1.DefaultHPAUpscaleDelay},
+	}
+
+	if k.config.HorizontalPodAutoscalerConfig != nil {
+		if k.config.HorizontalPodAutoscalerConfig.CPUInitializationPeriod != nil {
+			horizontalPodAutoscalerConfig.CPUInitializationPeriod = k.config.HorizontalPodAutoscalerConfig.CPUInitializationPeriod
```

Comment:
```suggestion
		if v := k.config.HorizontalPodAutoscalerConfig.CPUInitializationPeriod; v != nil {
			horizontalPodAutoscalerConfig.CPUInitializationPeriod = v
```

This pattern makes it a bit shorter